doc/api/fs.md

```diff
@@ -3205,11 +3205,15 @@ changes:
   * `encoding` {string|null} **Default:** `null`
   * `flag` {string} See [support of file system `flags`][]. **Default:** `'r'`.
   * `signal` {AbortSignal} allows aborting an in-progress readFile
+  * `kPerRead` {integer} The number of kb per read, -1 means no limit.
```

Comment:
The options are in alphabetical order. Let's keep them that way by moving this up one line.

Reply:
```suggestion
  * `kPerRead` {integer} The number of KB per read. Use `-1` for no limit.
```

Reply:
Is there a reason why we're using kilobytes instead of just bytes?

Also, I think a name like `chunkSize` or similar would be a better name, especially since the `/^k[A-Z]/` prefix is used internally in code for private/internal field/symbol names.